CONTRIBUTING.rst

```diff
@@ -6,6 +6,43 @@ so do not hesitate!
 
 Contributions can be made in the form of feature requests, bug reports and feedback.
 
+First Time
+----------
+
+- Download and install the `latest version of git`_.
+- Configure git with your `username`_ and `email`_::
+
+        git config --global user.name 'your name'
+        git config --global user.email 'your email'
+
+- Make sure you have a `GitHub account`_.
+- Fork Bottery to your GitHub account by clicking the `Fork`_ button.
+- `Clone`_ your GitHub fork locally::
+
+        git clone https://github.com/{username}/bottery/
+        cd bottery
+
+- Add the main repository as a remote to update later::
+
+        git remote add rougeth https://github.com/rougeth/bottery
+        git fetch rougeth
+
+- Create a virtualenv::
+
+        python3 -m venv env
+        . env/bin/activate
+        # or "env\Scripts\activate" on Windows
```

Comment:
`installation.rst` has a good text guide on how to install and use virtualenv, with nice references, we could create a link here, right?

Reply:
Yes, for sure!